src/protobuf-net.AspNetCore/protobuf-net.AspNetCore.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
tabs!?

Reply:
Hey, you can blame the latest few devenv previews for that. It *really, really* wants to change csproj to tabs any time you so much as glance at it. There's only so many times I can keep deleting tabs and manually adding spaces.